### DISCUSSION TITLE
What should we call the .NET 8 compatible version of the library?

### DISCUSSION THREAD

**Discussion by adrianhall** [Azure/azure-mobile-apps#546] [Polls]
Hey folks,

.NET 8 is coming right around the corner.  Since this is an LTS release, I will be snapping all the libraries to the new version.  (Don't worry, I'll still be supporting .NET 6).  I am seeking input on what the new version of the library should be.

**Comment by henda79** [comment]
Yes, I think 8.0

What happened to 7, it seems like there's a new .NET every month. I'm not even onto 6 / MAUI yet.

**Comment by fileman** [comment]
I think is more clear if main release number match .net version
example 8.x.y releases are based on .net 8
